python/mxnet/libinfo.py

```diff
@@ -29,6 +30,17 @@ def find_lib_path():
     lib_path : list(string)
         List of all found path to the libraries.
     """
+    lib_from_env = os.environ.get('MXNET_LIBRARY_PATH')
+    if lib_from_env:
```

Comment:
Does this also check for an empty string?  Some OS's, at least in C, will return a valid pointer from getenv() for an empty, yet defined, string
ie
export MXNET_LIBRARY_PATH=
will return "" on some OS's, even though the intent is clearly to not be defined.

Reply:
Yes it does

Reply:
so which code path will be taken for an empty string?

Reply:
The default one below (starting with ``curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))``)

Reply:
I don't see any evidence to support that claim, but whatever -- I don't care all that much.

Reply:
Just open the code and look below the introduced if-block :D

Reply:
it continue in line 32/44 as before - we just inserted special handling in case that variable is set

Reply:
I understand, however I think the following code will take the if branch, which it shouldn't:
```python
lib_from_env = "
if lib_from_env:
   print(lib_from_env)
```

Reply:
No it won't. I just double checked.

Reply:
ok

Reply:
thanks for checking